### PR TITLE
fix(pdf): upgrade pdfjs to 3.6.172

### PR DESCRIPTION
**Pull request by bfoxx1906**
fixes issue with client custom conversion issue causing text layer on pdf's to not be visible

### CODE REVIEW

File: src/lib/viewers/controls/controls-root/ControlsRoot.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
     position: absolute;
     bottom: 25px;
     left: 50%;
+    z-index: 3;
```
Comment (jstoffan) [inline src/lib/viewers/controls/controls-root/ControlsRoot.scss:7]: Why is this needed?

Comment (bfoxx1906) [reply src/lib/viewers/controls/controls-root/ControlsRoot.scss:7]: because the text layer z-index was set to 2 in this release and it was covering the buttons

Comment (bfoxx1906) [reply src/lib/viewers/controls/controls-root/ControlsRoot.scss:7]: because the text layer z-index was set to 2 in this release and it was covering the buttons

Comment (bfoxx1906) [reply src/lib/viewers/controls/controls-root/ControlsRoot.scss:7]: because the text layer z-index was set to 2 in this release and it was covering the buttons

**Review by JChan106 — APPROVED** [review]
